src/main/java/uk/gov/hmcts/darts/common/repository/CaseRetentionRepository.java

```diff
@@ -19,10 +19,10 @@ public interface CaseRetentionRepository extends JpaRepository<CaseRetentionEnti
 
     @Query("""
         SELECT c
-        FROM CaseRetentionEntity c
-        WHERE courtCase = :courtCase
-        AND manualOverride=false
-        AND currentState='COMPLETE'
+        FROM CaseRetentionEntity c, RetentionPolicyTypeEntity rpt
```

Comment:
If you're navigating across the entities on line 24, then why would you need to add rpt here?

Reply:
removed